CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# CLIMADA Contribution Guide
+
+Thank you for contributing to CLIMADA!
+
+## Overview
+
+Before you start, please have a look at our [Developer Guide][devguide].
+
+To contribute follow these steps:
+
+1. Fork the project on GitHub.
+2. Create a local clone of the develop branch (`git clone https://github.com/YOUR-USERNAME/climada_python.git -b develop`)
+3. Install the packages in `climada_python/requirements/env_climada.yml` and `climada_python/requirements/env_developer.yml`.
```

Comment:
There are already installation instructions in the doc, I think we should not duplicate them here in case something changes.